scripts/pre_commit_linter.py

```diff
@@ -44,12 +44,11 @@
 
 __author__ = "Barnabas Makonda(barnabasmakonda@gmail.com)"
 
+import sys
+import time
 import argparse
 import os
 import subprocess
-import sys
-import time
```

Comment:
OK -- let's work through this in a separate pass, when we figure out which rules should be enforced and which shouldn't.